### ISSUE TITLE
test_ip_splitter_remove_prefix_larger_than_input_range fails with python 3.7

### ISSUE THREAD

**Issue by Mic92** [netaddr/netaddr#182]
Full build log: https://gist.github.com/Mic92/ed6eaf2540634623d4a1e36cc0fd41bc

```

=================================== FAILURES ===================================
____________ test_ip_splitter_remove_prefix_larger_than_input_range ____________

self = IPNetwork('172.24.0.0/16'), prefixlen = 15, count = 1, fmt = None

    def subnet(self, prefixlen, count=None, fmt=None):
        """
        A generator that divides up this IPNetwork's subnet into smaller
        subnets based on a specified CIDR prefix.

        :param prefixlen: a CIDR prefix indicating size of subnets to be
            returned.

        :param count: (optional) number of consecutive IP subnets to be
            returned.

        :return: an iterator containing IPNetwork subnet objects.
        """
        if not 0 <= self.prefixlen <= self._module.width:
            raise ValueError('CIDR prefix /%d invalid for IPv%d!' \
                % (prefixlen, self._module.version))

        if not self.prefixlen <= prefixlen:
            #   Don't return anything.
>           raise StopIteration
E           StopIteration

netaddr/ip/__init__.py:1261: StopIteration

The above exception was the direct cause of the following exception:

    def test_ip_splitter_remove_prefix_larger_than_input_range():
        s = SubnetSplitter('172.24.0.0/16')
        assert s.available_subnets() == [IPNetwork('172.24.0.0/16')]
>       assert s.extract_subnet(15, count=1) == []

netaddr/tests/ip/test_ip_splitter.py:73:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
```

Noticed while upgrading python3.7 in nixpkgs.

**Comment by stmcginnis** [comment]
Looks like this is being addressed in https://github.com/drkjam/netaddr/pull/167

**Comment by natemcmaster** [comment]
I see #167 has been merged by @snordhausen. Is this going to be released anytime soon? Thanks :)

**Comment by jstasiak** [comment]
Fixed in 0.7.20.
